api/v1alpha1/driver_types.go

```diff
@@ -273,7 +273,7 @@ type DriverSpec struct {
 	// A reference to a ConfigMap resource holding image overwrite for deployed
 	// containers
 	//+kubebuilder:validation:Optional
-	//+kubebuilder:validation:XValidation:rule=self.?name.orValue("") != "",message="'.name' cannot be empty"
+	//+kubebuilder:validation:XValidation:rule=self.name != "",message="'.name' cannot be empty"
```

Comment:
@Madhu-1 Selef might not exist and be nil, that will throw the validation check.

Reply:
can you specify what exact case doesn't work with sample yamls. when i tested it worked as expected

Reply:
Checked the new rule with `self = nil` on both 1.29 and 1.30 and it works.